src/styles/components.scss

```diff
@@ -8,6 +8,10 @@
   @apply mx-auto max-w-[1202px] px-3;
 }
 
+form { 
```

Comment:
Are you sure you want to apply this to all forms across the website?
Can we use mx-auto on the form instead?

Reply:
I'm not sure if we'd have multiple form. Do you want me to change it?